<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>983</width>
    <height>724</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calaos Tools</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/img/logo_huge.png</normaloff>:/img/logo_huge.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QStackedWidget" name="Pages">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="pageProg">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="FormRules" name="widgetRules" native="true"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="pageDALI">
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="FormDali" name="widgetDali" native="true"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="pageAbout">
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QPushButton" name="pushButtonBack">
               <property name="text">
                <string>Retour</string>
               </property>
               <property name="icon">
                <iconset resource="resources.qrc">
                 <normaloff>:/img/go-previous.png</normaloff>:/img/go-previous.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTextEdit" name="textEdit">
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="midLineWidth">
              <number>-5</number>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
             <property name="html">
              <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;table border=&quot;0&quot; style=&quot;-qt-table-type: root; margin-top:4px; margin-bottom:4px; margin-left:4px; margin-right:4px;&quot;&gt;
&lt;tr&gt;
&lt;td style=&quot;border: none;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'DejaVu Sans';&quot;&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:12pt; font-weight:600; color:#cccccc;&quot;&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:12pt; font-weight:600; color:#cccccc;&quot;&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:12pt; font-weight:600; color:#cccccc;&quot;&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:12pt; font-weight:600; color:#cccccc;&quot;&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:20pt; font-weight:600; color:#cccccc;&quot;&gt;Calaos Tools&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#cccccc;&quot;&gt;Copyright (c) Calaos&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; color:#cccccc;&quot;&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-style:italic; color:#cccccc;&quot;&gt;version: 0.2&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-style:italic; color:#cccccc;&quot;&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; color:#cccccc;&quot;&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; color:#cccccc;&quot;&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; color:#cccccc;&quot;&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#cccccc;&quot;&gt;Visit our website: &lt;/span&gt;&lt;a href=&quot;http://www.calaos.fr&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#80a2cc;&quot;&gt;www.calaos.fr&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:9pt; text-decoration: underline; color:#0000ee;&quot;&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;img src=&quot;:/img/calaos.png&quot; /&gt;&lt;/p&gt;&lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>983</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionNouveau_projet"/>
    <addaction name="separator"/>
    <addaction name="actionCharger_un_projet"/>
    <addaction name="actionOuvrir_un_projet_en_ligne"/>
    <addaction name="separator"/>
    <addaction name="actionSauvegarder"/>
    <addaction name="actionSauvegarder_un_projet"/>
    <addaction name="actionSauvegarder_un_projet_en_ligne"/>
    <addaction name="separator"/>
    <addaction name="actionG_n_rer_fichier_d_impression"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuA_propos">
    <property name="title">
     <string>Aide</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionA_propos_de_Qt"/>
   </widget>
   <widget class="QMenu" name="menuProjet">
    <property name="title">
     <string>Projet</string>
    </property>
    <addaction name="actionSe_connecter"/>
    <addaction name="actionSe_d_connecter"/>
    <addaction name="separator"/>
    <addaction name="actionProgrammer_l_automate"/>
   </widget>
   <widget class="QMenu" name="menuOutils">
    <property name="title">
     <string>Outils</string>
    </property>
    <addaction name="actionDALI"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuProjet"/>
   <addaction name="menuOutils"/>
   <addaction name="menuA_propos"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/exit.png</normaloff>:/img/exit.png</iconset>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/help.png</normaloff>:/img/help.png</iconset>
   </property>
   <property name="text">
    <string>A propos</string>
   </property>
  </action>
  <action name="actionA_propos_de_Qt">
   <property name="text">
    <string>A propos de Qt</string>
   </property>
  </action>
  <action name="actionCharger_un_projet">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/document-open.png</normaloff>:/img/document-open.png</iconset>
   </property>
   <property name="text">
    <string>Ouvrir un projet...</string>
   </property>
  </action>
  <action name="actionSauvegarder_un_projet">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/document-save.png</normaloff>:/img/document-save.png</iconset>
   </property>
   <property name="text">
    <string>Sauvegarder un projet sous...</string>
   </property>
  </action>
  <action name="actionNouveau_projet">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/document-new.png</normaloff>:/img/document-new.png</iconset>
   </property>
   <property name="text">
    <string>Nouveau projet...</string>
   </property>
  </action>
  <action name="actionSauvegarder">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/document-save.png</normaloff>:/img/document-save.png</iconset>
   </property>
   <property name="text">
    <string>Sauvegarder</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSe_connecter">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/user-online.png</normaloff>:/img/user-online.png</iconset>
   </property>
   <property name="text">
    <string>Se connecter</string>
   </property>
  </action>
  <action name="actionSe_d_connecter">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/user-invisible.png</normaloff>:/img/user-invisible.png</iconset>
   </property>
   <property name="text">
    <string>Se déconnecter</string>
   </property>
  </action>
  <action name="actionProgrammer_l_automate">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/system-software-update.png</normaloff>:/img/system-software-update.png</iconset>
   </property>
   <property name="text">
    <string>Programmer l'automate</string>
   </property>
  </action>
  <action name="actionProgrammer_le_serveur">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/system-software-update.png</normaloff>:/img/system-software-update.png</iconset>
   </property>
   <property name="text">
    <string>Programmer le serveur</string>
   </property>
  </action>
  <action name="actionOuvrir_un_projet_en_ligne">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/go-first.png</normaloff>:/img/go-first.png</iconset>
   </property>
   <property name="text">
    <string>Ouvrir un projet en ligne...</string>
   </property>
  </action>
  <action name="actionSauvegarder_un_projet_en_ligne">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/go-last.png</normaloff>:/img/go-last.png</iconset>
   </property>
   <property name="text">
    <string>Sauvegarder un projet en ligne...</string>
   </property>
  </action>
  <action name="actionDALI">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/icon_light_on.png</normaloff>:/img/icon_light_on.png</iconset>
   </property>
   <property name="text">
    <string>DALI...</string>
   </property>
  </action>
  <action name="actionG_n_rer_fichier_d_impression">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/img/edit-rename.png</normaloff>:/img/edit-rename.png</iconset>
   </property>
   <property name="text">
    <string>Générer fichier d'impression...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>FormDali</class>
   <extends>QWidget</extends>
   <header>formdali.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FormRules</class>
   <extends>QWidget</extends>
   <header>formrules.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
